source/sass/wagtail/profiles.scss

```diff
@@ -26,6 +26,19 @@
     grid-template-columns: 0.9fr 2fr;
     min-height: 160px;
   }
+
+  &-sans-location {
```

Comment:
Let's please not do this, it makes it _really hard_ to find CSS rules. If it's a second class, then using `&.someclass` is fine, but if it's literally only part of the name, then this is going to make it virtually impossible to find anything, because people are going to be searching for the full class as found in the DOM inspector.

Reply:
I completely understand your point here. 👍